package.json

```diff
@@ -61,7 +61,7 @@
     "styled-system": "^5.1.2"
   },
   "devDependencies": {
-    "@applitools/eyes-storybook": "3.2.2",
```

Comment:
this is locked to 3.2.2 because later versions seem to stop working and I haven't bothered to debug why